Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm

```diff
@@ -1113,11 +1113,12 @@ macro preOp(opcodeName, opcodeStruct, integerOperation)
 end
 
 
-llintOpWithProfile(op_to_number, OpToNumber, macro (size, get, dispatch, return)
+llintOpWithReturn(op_to_number, OpToNumber, macro (size, get, dispatch, return)
     get(m_operand, t0)
     loadConstantOrVariable(size, t0, t2, t3)
     bieq t2, Int32Tag, .opToNumberIsInt
```

Comment:
Don't you need to update the profile with `ArithProfileInt` in this case?

Reply:
Actually this is not necessary. And we are avoiding that since this is too costly for the major case.